builder-base/github-cli-amd64-checksum

```diff
@@ -1 +1 @@
-41b879bc3c1c8304c10ddd5e0c1d8d40aaa97524fb17f832471b3f01f6909922  gh_1.8.0_linux_amd64.tar.gz
+41b879bc3c1c8304c10ddd5e0c1d8d40aaa97524fb17f832471b3f01f6909922 gh_1.8.0_linux_amd64.tar.gz
```

Comment:
We need 2 spaces here (and in some files below). It denotes text mode for the sha256sum command.

Reply:
ah ok